src/waveform/renderers/waveformrenderbeat.cpp

```diff
@@ -22,24 +22,38 @@ WaveformRenderBeat::~WaveformRenderBeat() {
 void WaveformRenderBeat::setup(const QDomNode& node, const SkinContext& context) {
     m_beatColor.setNamedColor(context.selectString(node, "BeatColor"));
     m_beatColor = WSkinColor::getCorrectColor(m_beatColor).toRgb();
+    m_barColor.setNamedColor(context.selectString(node, "BarColor"));
+    m_barColor = WSkinColor::getCorrectColor(m_barColor).toRgb();
+    m_phraseColor.setNamedColor(context.selectString(node, "PhraseColor"));
+    m_phraseColor = WSkinColor::getCorrectColor(m_phraseColor).toRgb();
 }
 
 void WaveformRenderBeat::draw(QPainter* painter, QPaintEvent* /*event*/) {
     TrackPointer trackInfo = m_waveformRenderer->getTrackInfo();
 
+    // No track, nothing to do
     if (!trackInfo)
         return;
 
+    // Gets beats from the track
```

Comment:
This comment does not add extra info. 
I think it can be removed.

Reply:
Done